courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -77,68 +76,58 @@ A Little "Preaching" About Names
    function Square_Root (V: Real) return Real;
    function Is_Open (V: Valve) return Boolean;
 
-========================
-Declarations and Bodies
-========================
-
-----------
-Examples
-----------
-
-.. include:: examples/070_subprograms/declarations_and_bodies.rst
+========
+Syntax
+========
 
-:url:`https://learn.adacore.com/training_examples/fundamentals_of_ada/070_subprograms.html#declarations-and-bodies`
+-------------------------
+ Specification and Body
+-------------------------
 
---------------------------
-Subprogram Declarations
---------------------------
+* Subprogram specification is the external (user) **interface**
 
-* Define the external (user) interface
+   - **Declaration** and **specification** are used synonymously
 
-   - Provide the subprogram specification
-   - *Declaration* and *specification* are often used synonymously
+* Specification may be required in
 
-* Required in some circumstances
+   - Package specification
+   - Recursion
+   - (Optional otherwise)
```

Comment:
There is at least two other non-optional (private child subprograms, generic subprograms). I would either complete the list (and I'm not sure I know what the full list would be) or just leave the original (perhaps removing "You'll see them")

Reply:
We provide example later in the slide for those two. So I'm adding back the original, but on top of them